### DISCUSSION TITLE
에디터 서버 오류

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#34] [Issue Report (Legacy)]
12월 4일 15시 이후 현재까지 유니티 에디터에서 서버 연결이 안됩니다.
이전에 제작한 테스트 링크의 경우는 접속이 잘되며 맥과 윈도우 같은 문제가 나타나고 있습니다

<img width="795" alt="스크린샷 2021-12-04 오후 4 57 14" src="https://user-images.githubusercontent.com/90679926/144702287-b6749743-7d73-4354-b4c6-939663679c1f.png">

**Comment by x4cf** [comment]
12월 5일 정상작동함

**Comment by hyeonjin-jo** [comment]
안녕하세요

현재는 재현이 안되고 있는데 혹시나 Environments가 Development에서 동작하지 않는 문제가 발생한다면

https://studio.zepeto.me/guides/accessing-to-multiplay-ip-addresses <- 가이드 문서의 Custom Environment를 만드는 방법으로 임시로
진행이 가능합니다

혹시 내부에 다른 문제가없는지는 살펴보도록 하겠습니다

감사합니다

![스크린샷 2021-12-06 오후 12 14 49](https://user-images.githubusercontent.com/92719846/144781398-ecf40e7a-07f7-4b73-9fbb-48d5399fe7ce.png)
